New package: limine-4.20230514.0.

mobinmob opened the pull request:
In this first release, `--enable-uefi-cd` is missing, because in my tests it breaks the build. I hope I can enable it in a future spkgbuild release :)

mobinmob committed:
New package: limine-4.20230514.0.

Visone-Selektah commented:
It build for me with `--enable-all` or with the option `--enable-efi-cd` added

mobinmob commented:
> It build for me with `--enable-all` or with the option `--enable-efi-cd` added

Really?
It did not for me (in a chroot). Let me try again...
I am not sure --enable-all makes much sense for venom, but I will try to see the results.

mobinmob committed:
limine: enable limine-cd-efi.bin build.

mobinmob commented:
> It build for me with `--enable-all` or with the option `--enable-efi-cd` added

I added  `--enable-efi-cd ` , it works. `--enable-all `created a uefi binary for aarch64 also I see...

lumarogit commented:
Hi !!!  clang and lld contains llvm as dep, you can remove it from depends to avoid redundant deps.

mobinmob committed:
Update spkgbuild

remove llvm from # dependencies, it is already a dependency though clang and lld - thanks @lumarogit!

mobinmob commented:
> Hi !!! clang and lld contains llvm as dep, you can remove it from depends to avoid redundant deps.

Thank you, done :)
